core/src/com/unciv/models/stats/Stats.kt

```diff
@@ -148,9 +148,9 @@ open class Stats(
     fun applyRankingWeights() {
         food *= 14
         production *= 12
-        gold *= 8 // 3 gold worth about 2 production
+        gold *= 6 // 2 gold worth about 1 production
```

Comment:
The AI generally has a problem with obtaining gold. This is partly because there aren't many gold opportunities early on. But reducing the value of gold wouldn't help.

Reply:
It's not just the AI who uses AI citizen assignment, but also many humans do it. In the early game, the most efficient use of gold is via unit upgrades, which is a gold:production conversion of 2:1 Late in the game, with stock exchanges built, the conversion of base gold to base production is more like 2*(1+0.1+0.1+0.15+0.2) : 1*(1+0.25+0.25+0.25) = 1.77:1, or even 1.19:1 with Professional Army/Pentagon. It makes sense to scale the value of gold with era, but I'm not mistaken the AI already takes modifiers from buildings into account.

Reply:
The reason AI loses as much gold as it does, got more to do with how it goas about picking city constructions and building roads without connecting cities. Improving their worker management might help quite a bit, but I'm not sure how to do so.